gateway/mw_jwt_test.go

```diff
@@ -45,6 +46,8 @@ jQIDAQAB!!!!
 -----END PUBLIC KEY-----
 `
 
+const invalidSignToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"
```

Comment:
`invalidSignToken` is specific to `TestJWTSessionFailRSA_WrongJWT_Signature`. We can move it to the test.